src/components/TestDeeplinkHandler.tsx

```diff
@@ -0,0 +1,43 @@
+import { useInitializeWallet } from '@/hooks';
+import { logger } from '@/logger';
+import { Navigation } from '@/navigation';
+import Routes from '@/navigation/routesNames';
+import { useEffect } from 'react';
+import { Linking } from 'react-native';
+import URL from 'url-parse';
+
+export function TestDeeplinkHandler() {
```

Comment:
could you add a comment here to clarify its purpose - I keep thinking it's something to do with testing deeplink handling; can add a comment even just referring to the line on the e2e/README file regarding it